README.md

```
@@ -150,6 +151,10 @@ All eight adapters detect the same phrases and suggest the same alternatives. Ea
 | Reviewdog runner | [reviewdog-no-animal-violence](https://github.com/Open-Paws/reviewdog-no-animal-violence) | Inline annotations on PR diffs |
 | Danger.js plugin | [danger-plugin-no-animal-violence](https://github.com/Open-Paws/danger-plugin-no-animal-violence) | Consolidated PR review comment |
 
+## Code Quality
+
+<img src="scorecard.png" width="100%">
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)